app/js/script.js

```diff
@@ -0,0 +1,173 @@
+let currentTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+let manualControlMode = null;
+let intervalId = null;
+let isPaused = false;
+let customTime = null;
+
+const timeZones = [
+  { name: "Local Time", value: currentTimeZone },
+  { name: "UTC", value: "UTC" },
+  { name: "New York (EST)", value: "America/New_York" },
+  { name: "Los Angeles (PST)", value: "America/Los_Angeles" },
+  { name: "London (GMT)", value: "Europe/London" },
+  { name: "Tokyo (JST)", value: "Asia/Tokyo" },
+];
+
+const manualControls = [
+  { name: "Restart a clock" },
+  { name: "Stop to set a time" },
+];
+
+function setCurrentTime() {
+  if (manualControlMode === "Stop to set a time") return;
+
+  let currentTime = customTime ? new Date(customTime) : new Date();
+  if (!customTime) {
+    currentTime = new Date(
+      currentTime.toLocaleString("en-US", { timeZone: currentTimeZone })
+    );
+  }
+
+  const seconds = currentTime.getSeconds();
+  const minutes = currentTime.getMinutes();
+  const hours = currentTime.getHours();
+
+  const timeInterval = 6;
+
+  const secondHand = seconds * timeInterval;
+  const minuteHand = minutes * timeInterval + seconds / 10;
+  const hourHand = hours * 30 + minutes / 2;
+
+  document.querySelector("#hour").style.transform = `rotate(${hourHand}deg)`;
+  document.querySelector(
+    "#minute"
+  ).style.transform = `rotate(${minuteHand}deg)`;
+  document.querySelector(
+    "#second"
+  ).style.transform = `rotate(${secondHand}deg)`;
+
+  if (customTime) {
+    customTime.setSeconds(customTime.getSeconds() + 1);
+  }
+}
+
+function startClock() {
+  setCurrentTime();
+  intervalId = setInterval(setCurrentTime, 1000);
+}
+
+function stopClock() {
+  clearInterval(intervalId);
+}
+
+startClock();
```

Comment:
Function call is probably starting a normal clock workflow when the application is launched, so I'd  suggest moving it to the end of the file, after all declarations.